src/aws/nist-800-53-rev4/rules/aws-nist-800-53-rev4-3.1.ts

```diff
@@ -52,6 +54,6 @@ export default {
   severity: 'high',
   conditions: {
     path: '@.kmsKeyId',
-    notEqual: null,
+    isEmpty: false,
```

Comment:
what requires this fix? Does the kmsKeyId get set to empty string if its not there?

Reply:
this was just a general improvement for robustness.. and Dgraph may process empty value different than our internal format (ie. maybe if we send undefined to Dgraph, it returned it back as null in the query), with braoder checks, we don't need to deal with all those cases... if no kmsKeyId then fail (be it '', null, undefined, false, or even 0!)